src/styles/main.scss

```diff
@@ -1,3 +1,238 @@
-body {
+$color-main-accent: #060b35;
+
+$color-secondary: #616070;
+
+$color-accent-blue: #00acdc;
+
+* {
+  box-sizing: border-box;
+}
+
+//========== Body, Section, Row =========== //
+.body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
+.catalog {
```

Comment:
check the checklist
<img width="666" alt="image" src="https://user-images.githubusercontent.com/102411130/213122209-9ae36219-d9ba-4915-9aa0-f648c7464a09.png">